### PR TITLE
lvm2: 2.03.18 -> 2.03.19

### PR DESCRIPTION
**Pull request by ajs124**
###### Description of changes
See commit messages.

`pkgsMusl.lvm2` and `pkgsStatic.lvm2` still build on `x86_64-linux`, but I haven't run any extensive tests yet (not even all nixos tests).

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by flokli** [comment]
This should probably target staging, no?

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -934,6 +934,7 @@ mapAliases ({
   lua5_sec = throw "'lua5_sec' has been renamed to/replaced by 'luaPackages.luasec'"; # Converted to throw 2022-02-22
   lumo = throw "lumo has been removed: abandoned by upstream"; # Added 2022-04-25
   lumpy = throw "lumpy has been removed from nixpkgs, as it is stuck on python2"; # Added 2022-01-12
+  lvm2-2_02 = lvm2;
```
Comment (flokli) [inline pkgs/top-level/aliases.nix:937]: That lvm2 was only used to give a lvm in the static case, I doubt anyone has been adding `lvm2-2_02` explicitly.

I'd drop it alltogether without creating an alias.

Comment (ajs124) [reply pkgs/top-level/aliases.nix:937]: yeah, I wasn't sure if we should add an alias. I don't really have a strong opinion on adding it or not, so if you feel like we shouldn't have it, I can drop it.

Comment (flokli) [reply pkgs/top-level/aliases.nix:937]: 👍

**Review by 7c6f434c — APPROVED** [review]
Code looks good

**Comment by 7c6f434c** [comment]
`python3Packages.aiohttp` should be an unrelated staging problem, right?

**Comment by ajs124** [comment]
> `python3Packages.aiohttp` should be an unrelated staging problem, right?

I really hope so

**Pull request merged**